src/dsymbol/conversion/first.d

```diff
@@ -1148,9 +1148,7 @@ private:
 
 		foreach (suffix; type.typeSuffixes)
 		{
-			if (suffix.star != tok!"")
-				continue;
-			else if (suffix.type)
+			if (suffix.type)
 				lookup.breadcrumbs.insert(ASSOC_ARRAY_SYMBOL_NAME);
```

Comment:
would it be possible to store the AA key type in the breadcrumbs as well without breaking anything?

Reply:
That should be doable, but as mentioned in the other comment, this is out of scope for this PR, i'll attempt it in a separate PR